sdk/tests/core/test_az_logging.c

```diff
@@ -413,7 +366,7 @@ static void test_az_log_everything_on_null(void** state)
     _az_LOG_WRITE((az_log_classification)12345, AZ_SPAN_EMPTY);
     _az_LOG_WRITE(AZ_LOG_MQTT_RECEIVED_PAYLOAD, AZ_SPAN_EMPTY);
 
-    assert_int_equal(_az_BUILT_WITH_LOGGING(4, 0), _log_retry);
+    assert_int_equal(_az_BUILT_WITH_LOGGING(5, 0), _log_retry);
```

Comment:
I was reading this, and `_log_retry` was confusing, especially given that there are log classifications having `RETRY` in their name. I had to expand the code above to make sure it what it really counts what I think it should. Rename to (`_number_of``) `_log_attempts`/`_log_invocations`?